pkg/cmd/inletsoperator_app.go

```diff
@@ -34,11 +34,7 @@ func makeInstallInletsOperator() *cobra.Command {
 	inletsOperator.Flags().String("pro-client-image", "", "Docker image for inlets-pro's client")
 
 	inletsOperator.RunE = func(command *cobra.Command, args []string) error {
-		kubeConfigPath := getDefaultKubeconfig()
-
-		if command.Flags().Changed("kubeconfig") {
-			kubeConfigPath, _ = command.Flags().GetString("kubeconfig")
-		}
+		kubeConfigPath, _ := command.Flags().GetString("kubeconfig")
```

Comment:
So this doesn't seem to use `IsChanged()` or the method that tried to ascertain the best KUBECONFIG to use. 

I can't see any notes or detail on the commits, so please could you add a note explaining why that was removed and how it's been tested?

It'd also be good to get all these commits squashed down into one. 👍

Reply:
the logic of this was moved inside of the kubectl function this seems to be a left over and needs to be deleted.